### ISSUE TITLE
Stacks: add a line to a stack

### ISSUE THREAD

**Issue by johnwdubois** [johnwdubois/rezonator#11]
**Is your feature request related to a problem? Please describe.**
When in Stack mode, if you click on the space between two words, nothing happens. Also, if you click on the speaker label etc, nothing happens.

**Describe the solution you'd like**
When in Stack mode, if you click on the space between two words, it should add the line to the current stack. 
In the same way, if you click on the speaker label etc, it should add the line to the current stack. 

**Describe alternatives you've considered**
Basically, if you click anywhere in the current line (on a word, on empty space, on the speaker label...), it should add the line to the current stack.

**Additional context**
Dragging your mouse over multiple speaker labels should add all of the lines you indicate to the current stack.
